src/Compilers/CSharp/Test/Emit/Emit/DynamicAnalysis/DynamicInstrumentationTests.cs

```diff
@@ -190,6 +190,104 @@ .maxstack  4
         }
 
         [Fact]
+        public void ArrowPropertyCoverage()
+        {
+            string source = @"
+using System;
+
+public class Program
+{
+    public static void Main(string[] args)
+    {
+        TestMain();
+    }
+
+    static void TestMain()
+    {
+        int x = Count;
+    }
+
+    static int Function(int x) => x;
```

Comment:
I added one, though those don't actually fall into the set of issues anyway (because lambdas are not treated as separate methods for dynamic analysis).